### PR TITLE
added isort installation

### PR DESCRIPTION
**Pull request by cenouralm**

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -20,6 +20,7 @@
 
 tests_require = [
     'pytest-invenio>=1.3.2',
+    'isort>=4.2.2',
```
Comment (fenekku) [inline setup.py:23]: pytest-invenio provides `isort` already... What was the issue here?

Comment (cenouralm) [reply setup.py:23]: I was trying to run "./run-tests" but it kept saying "isort: command not found". After I added this line and running " pip install -e .[all]" it worked fine.

Comment (cenouralm) [reply setup.py:23]: Just tested and it worked only running `pip install -e .[all]`.

Comment (fenekku) [reply setup.py:23]: Ok. Closing.

**Pull request closed without merging**